docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md

```diff
@@ -0,0 +1,73 @@
+---
+id: assistant-plugin
+title: Adding Assets via the Assistant plugin
+sidebar_label: Assistant Plugin
+---
+
+## ​Create a New Library
+
+To create a new library from the Assistant Plugin:
+
+* Open the Libraries App
+* Click the plus symbol (+)
+* Enter a name for your library
+* Click the Enter button or hit Enter/Return on your keyboard 
+
+![Create library via Assistant plugin](/img/assistant/cloud--libraries--assistant-plugin--1.jpg)
+
+## Add Items to a Library
+
+You can add page content like posts, pages, Beaver Builder Templates, etc. from the [Content App](../../../plugin/apps/content.md), [Media App](../../../plugin/apps/media.md), or the [Libraries App](../../../plugin/apps/libraries.md).
+
+## Add Content via the Content App
+
+To add content from the Content App, click the piece of content you want to add, then select the Add to Library button.  You’ll be prompted to select which library you’d like to add the content to.
+
+<video autoPlay loop>
+<source src="/video/assistant/libraries--add-assets-content-app.mp4" type="video/mp4" />
+<source src="/video/assistant/libraries--add-assets-content-app.webm" type="video/webm" />
+</video>
+
+## Add Content via the Media App
+
+You can add .jpg, .png, and .svg graphics to Assistant Pro from the Media App.  By default, you can’t add .svg graphics to WordPress media storage, so being able to add .svg graphics to WordPress via the Assistant Plugin can save you the trouble of using ftp to add .svg graphics to your WordPress sites.
```

Comment:
This is repeating info from elsewhere, and it appears to have left out .gif. Better to use a link.

Reply:
By default....

NH: This seems like a sales feature so I'd recommend making a tip out of it.

Reply:
Fixed `.gif`